tools/stern/README.md

```diff
@@ -0,0 +1,12 @@
+Stern - Backoffice facade
```

Comment:
You mention 'backoffice' in the description; but don't really describe what a 'backoffice' is or what it does; I'd say either remove references to backoffice and just provide the description of functionality; or define what a backoffice is 😄

Reply:
+1.

"Back office" is technically an existing term (https://en.wikipedia.org/wiki/Front_and_back_office_application) but it's so vague that I'd rather have a better description there.

Reply:
_...such as enterprise resource planning (ERP), inventory control, manufacturing and all of the supply chain activities associated with procuring goods, services and raw materials..._ 👌

Reply:
Oh wait, wrong link. Nevermind